rs/crypto/test_utils/tls/src/lib.rs

```diff
@@ -62,13 +62,18 @@ impl From<&CipherSuite> for rustls::SupportedCipherSuite {
 pub fn temp_crypto_component_with_tls_keys(
     registry: Arc<FakeRegistryClient>,
     node_id: NodeId,
+    with_remote_vault: bool,
```

Comment:
Why the remote vault?

Reply:
It was starting a new runtime inside the test runtime, causing a panic. Fixed by separating the runtimes